modules/docker/README.md

```diff
@@ -0,0 +1,187 @@
+- `dkVls` List volumes
+- `dkVpr` Remove all unused volumes
+- `dkVrm` Remove one or more volumes
+
+#### network (N)
+
+- `dkN` Manage networks
+- `dkNs` Connect a container to a network
+- `dkNx` Disconnects a container from a network
+- `dkNin` Displays detailed information on a network
+- `dkNls` Lists all the networks created by the user
+- `dkNpr` Remove all unused networks
+- `dkNrm` Deletes one or more networks
+
+#### system (Y)
+
+- `dkY` Manage Docker
+- `dkYdf` Show docker filesystem usage
+- `dkYpr` Remove unused data
+
+#### stack (K)
+
+- `dkK` Manage Docker stacks
+- `dkKls` List stacks
+- `dkKps` List the tasks in the stack
+- `dkKrm` Remove the stack
+
+#### swarm (W)
+
+- `dkW` Manage Docker Swarm
+
+### Docker Machine
+
+- `dkm` is short for `docker-machine`
+- `dkma` Get or set the active machine
+- `dkmcp` Copy files between machines
+- `dkmd` Set up the default machine ; alowing you to use `dkme` without arguments
+- `dkme` Set up the environment for the Docker client (eg: `dkme staging` to toggle to staging)
+- `dkmin` Inspect information about a machine
+- `dkmip` Get the IP address of a machine
+- `dkmk` Kill a machine
+- `dkmls` List machines
+- `dkmpr` Re-provision existing machines
+- `dkmps` is alias for `dkmls`
+- `dkmrg` Regenerate TLS Certificates for a machine
+- `dkmrm` Remove a machine
+- `dkms` Start a machine
+- `dkmsh` Log into or run a command on a machine with SSH
+- `dkmst` Get the status of a machine
+- `dkmS` Restart a machine
+- `dkmu` Get the URL of a machine
+- `dkmup` Upgrade a machine to the latest version of Docker
+- `dkmV` Show the Docker Machine version or a machine docker version
+- `dkmx` Stop a machine
+
+### Docker Compose
+
+- `dkc` is short for `docker-compose`
+- `dkcb` Build or rebuild services
+- `dkcB` Build or rebuild services and do not use cache when building the image
+- `dkcd` Stop and remove containers, networks, images, and volumes
+- `dkce` Execute a command in a running container
+- `dkck` Kill containers
+- `dkcl` View output from containers
+- `dkcls` is alias for `dkcps`
+- `dkcp` Pause services
+- `dkcP` Unpause services
+- `dkcpl` Pull service images
+- `dkcph` Push service images
+- `dkcps` List containers
+- `dkcr` Run a one-off command
+- `dkcR` Run a one-off command and remove container after run.
+- `dkcrm` Remove stopped containers
+- `dkcs` Start services
+- `dkcsc` Set number of containers for a service
+- `dkcS` Restart services
+- `dkcu` Create and start containers
+- `dkcU` Create and start containers in detached mode:
+           Run containers in the background, print new container names
+- `dkcV` Show the Docker-Compose version information
+- `dkcx` Stop services
+
+## Author
+
+*The author of this module should be contacted via the [issue tracker][3].*
+
+- [François Vantomme](https://github.com/akarzim)
+
+[1]: https://www.docker.com/
+[2]: https://github.com/zplug/zplug
+[3]: https://github.com/akarzim/zsh-docker-aliases/issues
```

Comment:
This seems a bit confusing. We've started some discussions (see #38 and #33) about these blocks.

I'd like this to read something like this:

> ## Support
>
> If you're having problems, use the [Prezto issue tracker][].
>
> ## Acknowledgements
>
> This module is a copy of [akarzim/zsh-docker-aliases][] by [François Vantomme](https://github.com/akarzim) (MIT License).

[Prezto issue tracker]: https://github.com/zsh-users/prezto/issues
[akarzim/zsh-docker-aliases]: https://github.com/akarzim/zsh-docker-aliases

cc @johnpneumann @Cottser who have been part of those discussions